Danki2/Assets/Scripts/Camera/CameraShake.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraShake
+{
+    public const float DefaultInterval = 1 / 60f;
+    public float Strength { get; private set; }
+    public float Duration { get; private set; }
+    public float Interval { get; private set; }
+    private float intervalTimer;
```

Comment:
I think Frequency and frequencyTimer would be better names

Reply:
`Interval` represents time intervals for each movement within the shake. Frequency would be 1/Interval.

Reply:
Ah I see, in that case it'd be worth updating the parameter in `CameraShakeManager::AddCameraShake` from frequency to interval